### ISSUE TITLE
Unexpected behaviour when creating/converting trace instances

### ISSUE THREAD

**Issue by ppfeiff** [process-intelligence-solutions/pm4py#183]
Hi, I encountered an problem when creating and converting a EventLog to pandas. I'm using pm4py version 2.0.1.3 and pandas 1.1.4.

My code looks like this:
```
event_log = EventLog()

t0 = Trace()
t0.attributes["concept:name"] = 0

t1 = Trace()
t1.attributes["concept:name"] = 1

t2 = Trace()
t2.attributes["concept:name"] = 2

t3 = Trace()
t3.attributes["concept:name"] = 3

t4 = Trace()
t4.attributes["concept:name"] = 9999999

e0 = Event()
e1 = Event({"concept:name": "Activity 1"})
e2 = Event({"concept:name": "Activity 2"})
e3 = Event({"concept:name": "Activity 3"})
e4 = Event({"concept:name": "Activity 4"})
e5 = Event({"concept:name": "Activity 5"})

t1.append(e0)

t2.append(e2)

t3.append(e3)

t4.append(e4)

event_log.append(t0)
event_log.append(t1)
event_log.append(t2)
event_log.append(t3)
event_log.append(t4)

```
When I print the event_log or transform it to pandas using
```
print(convert_to_dataframe(event_log))
```
everything works fine.

However, when I want to append the same event to multiple traces, a weird behaviour can be seen.
E.g. (same events etc)
```
t1.append(e0)
t1.append(e1)

t2.append(e1)
t2.append(e2)

t3.append(e1)
t3.append(e2)
t3.append(e3)

t4.append(e1)
t4.append(e2)
t4.append(e3)
t4.append(e4)
t4.append(e5)

event_log.append(t0)
event_log.append(t1)
event_log.append(t2)
event_log.append(t3)
event_log.append(t4)
```

the event_logs still is fine, but the pandas dataframe has the same case:concept:name for almost all entries, except the first.
    case:concept:name concept:name
0                   1          NaN
1             9999999   Activity 1
2             9999999   Activity 1
3             9999999   Activity 2
4             9999999   Activity 1
5             9999999   Activity 2
6             9999999   Activity 3
7             9999999   Activity 1
8             9999999   Activity 2
9             9999999   Activity 3
10            9999999   Activity 4
11            9999999   Activity 5

I check the code. convert_to_dataframe first converts the EventLog to a EventStream. The EventStream is already broken so the issue migh occur there.

**Comment by Javert899** [comment]
Dear ppfeiff,

There is a parameter in the event stream converter that should overcome this problem (Parameters.DEEP_COPY), however it does not work appropriately.

We'll fix the problem in the next release

**Comment by fit-alessandro-berti** [comment]
Dear ppfeiff,

The issue is resolved in the version 2.1.0

The conversion should be called in the following way:

from pm4py.objects.conversion.log import converter as log_converter

stream = log_converter.apply(event_log, variant=log_converter.Variants.TO_EVENT_STREAM, parameters={log_converter.Variants.TO_EVENT_STREAM.value.Parameters.DEEP_COPY: True})
dataframe = log_converter.apply(stream, variant=log_converter.Variants.TO_DATA_FRAME)
print(dataframe)
